rocAL/rocAL_pybind/example/new_api/rocAL_api_python_unittest.py

```diff
@@ -55,11 +55,11 @@ def main():
     with pipe:
         jpegs, _ = fn.readers.file(file_root=data_path, shard_id=local_rank, num_shards=world_size, random_shuffle=True)
         images = fn.decoders.image(jpegs, file_root=data_path, device=decoder_device, output_type=types.RGB, shard_id=0, num_shards=1, random_shuffle=True)
-        images = fn.resize(images, device=rocal_device, resize_x=300, resize_y=300)
-
 
         if augmentation_name == "resize":
-            output = fn.resize(images, resize_x=300, resize_y=300)
+            output = fn.resize(images, device=rocal_device, resize_x=300, resize_y=300, scaling_mode=types.SCALING_MODE_NOT_SMALLER, interpolation_type=types.TRIANGULAR_INTERPOLATION)
```

Comment:
For default resize unit test do we need to use Not smaller mode and triangular interpolation?